sql-db/hibernate-reactive/src/main/java/io/quarkus/ts/reactive/BookDescription.java

```diff
@@ -0,0 +1,13 @@
+package io.quarkus.ts.reactive;
+
+import io.quarkus.hibernate.orm.panache.common.ProjectedFieldName;
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+@RegisterForReflection
+public class BookDescription {
+    public final String name;
+
+    public BookDescription(@ProjectedFieldName("title") String title) {
+        this.name = title;
+    }
+}
```

Comment:
please state -> why this class is needed?

Reply:
GENERAL: Please use field name `title` for book, not `name`

Reply:
This class is needed to test a DTO projection feature[1]. The name changed.

[1] https://quarkus.io/guides/hibernate-reactive-panache#query-projection